### PR TITLE
added example for consumer fetch

### PR DESCRIPTION
**Pull request by ajhunyady**

### CODE REVIEW

**Comment by sehz** [comment]
run `cargo fmt`

**Comment by ajhunyady** [comment]
? This is node.

**Comment by sehz** [comment]
sorry.￼

File: examples/fetch.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        // Explicitly call `.connect()` to connect to the cluster;
+        // This allows for lazily-loading the connection, useful in
+        // situations where the fluvio client does not need to immediately
+        // connect.
+        await fluvio.connect()
+
+        // Set the admin client to create and find topics;
+        const admin = await fluvio.admin()
+
+        // Create the topic
```
Comment (sehz) [inline examples/fetch.ts:38]: leave topic creation to CLI or admin API in separate flow.  just assume topic exist or verify topic

**Review by sehz — CHANGES_REQUESTED** [review]

File: examples/fetch.ts (hunk 1 above)
Comment (ajhunyady) [reply examples/fetch.ts:38]: This is a copy/paste from a consumer example, hence follows existing workflow. 
Needs overhaul by engineering for consistency.

**Review by sehz — APPROVED** [review]

**Pull request merged**